{"name": "test_LANDevice", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range", "trace": "self = <test_plugins.Test_plugins object at 0x000001ED39C990D0>\n\n    def test_LANDevice(self):\n        name1 = \"LANDevice\"\n        self.p2 = modify_conf_file()\n        self.p2.GetProperitesOfSubObjects()\n        self.p1.send_cmd(\"tftp -g -r testserver.conf 192.168.1.65\")\n        rel = self.p1.send_cmd(\"ls -l\")\n        self.p1.log(rel,name1,self.nowtime)\n        rel=self.p1.send_cmd(\"kill -USR1 `pidof testserver`\")\n        rel=self.p1.send_enter()\n        self.p1.log(rel,name1,self.nowtime)\n        rel = re.findall(r'\"Status\":\"(\\S)\"',rel)\n        print(rel)\n        print(type(rel))\n>       if rel[0] == \"0\":\nE       IndexError: list index out of range\n\ntest_plugins.py:57: IndexError"}, "attachments": [{"name": "log", "source": "2b784f32-b10c-4b5a-a026-a4fda50ff160-attachment.txt", "type": "text/plain"}], "start": 1700729675586, "stop": 1700729675718, "uuid": "12590c29-b86d-4435-a90b-6f1fdc2dd524", "historyId": "819dd07a0455106f3ec3b53725352a6f", "testCaseId": "819dd07a0455106f3ec3b53725352a6f", "fullName": "test_plugins.Test_plugins#test_LANDevice", "labels": [{"name": "suite", "value": "test_plugins"}, {"name": "subSuite", "value": "Test_plugins"}, {"name": "host", "value": "l29707B"}, {"name": "thread", "value": "12784-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_plugins"}]}